pages/programs/workforce/schedule.tsx

```diff
@@ -61,6 +61,20 @@ const CohortSchedule: NextPage = () => {
             </option>
           ))}
         </select>
+        {groupBy === 'cohort' && (
+          <select onChange={(e) => setFilter(e.target.value)} value={filter}>
+            {
+              <option value='' disabled={!filter} className={filter ? 'reset-option' : ''}>
+                {filter ? 'Reset' : 'Filter By Cohort'}
+              </option>
+            }
+            {filterOptions.map((id, i) => (
+              <option key={id + i} value={id}>
```

Comment:
Using `i` for the keys is an anti-pattern and shouldn't the ids be unique anyway?
```suggestion
              <option key={id} value={id}>
```

Reply:
using _**only**_ the index is. in this case, I believe it was because the `id` may show up more than once, at least, I'm pretty sure that's the only reason it would have been like that